src/action-sheets/index.js

```diff
@@ -587,3 +624,53 @@ export const showStreamActionSheet = ({
     }),
   );
 };
+
+export const showPmActionSheet = ({
```

Comment:
Really more like "PM *conversation* action sheet", right? For an individual PM, there's the message action sheet.

Reply:
(Similarly for the function constructing the buttons.)